README.russian.md

```diff
@@ -1,5 +1,7 @@
 # LuaConsole
-
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```

Comment:
I can't accept a change on this. This is automatically generated by a bot. Please revert this change.

Reply:
I don't understand how it can break something, but I'll revert this change